README.md

```diff
@@ -0,0 +1,16 @@
+# Welcome to Quarkiverse!
+
+The Quarkiverse GitHub organization provides repository hosting (including build, CI and release publishing setup) for Quarkus extension projects contributed by the community.
+
+You can browse the [Quarkiverse Wiki](https://github.com/quarkiverse/quarkiverse/wiki) to get more info.
+
+## Here is the flow to get your extension onboarded:
+
+1. The first step is to create or pick an **Extension Proposal** ticket in the [Quarkus issue tracker](https://github.com/quarkusio/quarkus/issues) and let us know if you are interested in leading the development of this extension by commenting on the issue. The Quarkus team will review and discuss with you and community if the proposal extension should belong to core or live in the Quarkiverse Hub.
+2. When decided, the Quarkiverse Team will initialize the repository in the Quarkiverse organization for you with skeleton code ready. Until then, if not already done, you can start implementing your extension from a skeleton with this command:
+```shell
+mvn io.quarkus:quarkus-maven-plugin:create-extension \
+ -DgroupId=io.quarkiverse.[myext] -DextensionId=[my-ext] -U
+```
+3. Enable in the [Quarkus Ecosystem-CI](https://github.com/quarkusio/quarkus-ecosystem-ci)
```

Comment:
What does that mean exactly?

Reply:
```suggestion
3. Follow instructions in [Quarkus Ecosystem-CI](https://github.com/quarkusio/quarkus-ecosystem-ci) to enable the extension to take part in overall testing.
```

Reply:
I think it means the steps to have your extension to build against Quarkus main branch: https://github.com/quarkusio/quarkus-ecosystem-ci#what-do-i-need-to-do-as-a-quarkus-platform-participant